src/app/components/accounts/consent-form/consent-form.component.ts

```diff
@@ -30,10 +31,10 @@ export class ConsentFormComponent implements OnInit {
         });
     }
 
-    onSubmit(FormData) {
+    onSubmit(FormData: unknown): void {
```

Comment:
`unknown` is recommended by ESLint to use in place of `any`

Reply:
form data has a specific type inferred from the form.

Reply:
the type of formData should be `FormGroup` and use `.get` to access its members.

Reply:
Okay I see that makes more sense.